crates/biome_cli/tests/snapshots/main_commands_explain/explain_valid_rule.snap

```diff
@@ -52,6 +52,18 @@ and the [the noopener documentation](https://html.spec.whatwg.org/multipage/link
 ```jsx
 <a href='http://external.link' target='_blank' rel="noopener" {...props}>child</a>
 ```
-```
 
+## Options
+
+Use the options to allow specific domains to use `target="_blank"` without `rel="noreferrer"`.
+
+```json
+{
+"//": "...",
+"options": {
+"allowDomains": ["example.com", "example.org"]
+}
+}
+```
+```
```

Comment:
``````suggestion

```json,ignore
{
	"//": "...",
		"options": {
		"allowDomains": ["https://example.com", "example.org"]
	}
}
```

```jsx,ignore
<>
	<a target="_blank" href="https://example.com"></a>
	<a target="_blank" href="example.org"></a>
<>
```

Biome doesn't check if the list contains valid URLs.

``````

I added some code snippets of how the valid JSX will look like using the options specified in the snippet. I also added a phrase at the end to set good expectations for the users. Does it look OK?

FYI I used `ignore` in the snippet, so the codegen won't try to evaluate the snippet.

Reply:
You can read more here: https://github.com/biomejs/biome/blob/main/crates/biome_analyze/CONTRIBUTING.md#document-the-rule

Reply:
Still getting assertion due to indentation with ignore, am I missing something?

![image](https://github.com/user-attachments/assets/10f59b04-f991-4b82-b2ac-8a393c0527c0)

Reply:
Sorry, I can't understand where it's coming from. Can you provide more context?